anyway/views/news_flash/api.py

```diff
@@ -131,6 +155,33 @@ def gen_news_flash_query(
     return query
 
 
+def gen_news_flash_query_new(query, valid_params: dict):
```

Comment:
I am not sure that the usage of trivial functions to set filters makes the code more maintainable. I myself would prefer to use direct filter when the filters are simple.